src/actionobserver.cpp

```diff
@@ -143,38 +143,43 @@ ActionObserver::ActionObserver(QObject *parent) :
     _actionObserver(new QMailActionObserver(this))
 {
     connect(_actionObserver, SIGNAL(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)),
-        this, SLOT(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)));
+            this, SLOT(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)));
 }
 
 // Report only long sync type of actions.
 // Small actions like exporting updates and flags are ignored, same for actions
 // that only happen on email client UI is visible(dowload inline images, message parts, ...)
 bool ActionObserver::isNotificationAction(QMailServerRequestType requestType)
 {
-    if (requestType == TransmitMessagesRequestType || requestType == RetrieveFolderListRequestType
-            || requestType == RetrieveMessageListRequestType || requestType == RetrieveMessagesRequestType
-            || requestType == RetrieveMessageRangeRequestType || requestType == RetrieveAllRequestType
-            || requestType == SynchronizeRequestType || requestType == RetrieveNewMessagesRequestType) {
-        return true;
-    }
-    return false;
+    return requestType == TransmitMessagesRequestType
+            || requestType == RetrieveFolderListRequestType
+            || requestType == RetrieveMessageListRequestType
+            || requestType == RetrieveMessagesRequestType
+            || requestType == RetrieveMessageRangeRequestType
+            || requestType == RetrieveAllRequestType
+            || requestType == SynchronizeRequestType
+            || requestType == RetrieveNewMessagesRequestType;
 }
 
 // ################ Slots #####################
 
 void ActionObserver::actionsChanged(QList<QSharedPointer<QMailActionInfo> > actionsList)
 {
-    foreach(QSharedPointer<QMailActionInfo> action, actionsList) {
+    foreach (QSharedPointer<QMailActionInfo> action, actionsList) {
```

Comment:
Whilst add this, this could be

```suggestion
    for (QSharedPointer<QMailActionInfo> action : actionsList) {
```

Reply:
Sure.